htdocs/emailcollector/class/emailcollector.class.php

```diff
@@ -2601,8 +2603,24 @@ public function doCollectOneCollector($mode = 0)
 										$errorforactions++;
 										$this->errors = $actioncomm->errors;
 									} else {
-										$operationslog .= '<br>Event created -> id='.dol_escape_htmltag($actioncomm->id);
+										if ($objectemail->status == Ticket::STATUS_CLOSED || $objectemail->status == Ticket::STATUS_CANCELED) {
```

Comment:
When we are here, email is not always related to a ticket.
You should be able to add a check on $fk_element_type to fix this

Also, as this change the behaviour, can you push this on develop branch.

Reply:
The pull request system is done this way.
If you made some change in the same local branch you develop the first initial PR, by just pushing the code on your own fork should also update the PR automatically.
So just try to fix on you local branch called 18.0 (using a new name of branch before pushing is a better practice) and it should be enough...

Reply:
As per your suggestion, I have updated the pull request with a Ticket check.